src/Evolution/Systems/RadiationTransport/M1Grey/Fluxes.hpp

```diff
@@ -0,0 +1,99 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"  // for item_type
+#include "DataStructures/DataBox/Prefixes.hpp"    // IWYU pragma: keep
+#include "DataStructures/Tensor/TypeAliases.hpp"  // IWYU pragma: keep
+#include "Evolution/Systems/GrMhd/ValenciaDivClean/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "Evolution/Systems/RadiationTransport/M1Grey/Tags.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "Utilities/Gsl.hpp"   // for not_null
+#include "Utilities/TMPL.hpp"  // for EXPAND_PACK_LEFT_TO...
+
+// IWYU pragma: no_include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+
+// IWYU pragma: no_forward_declare Tags::Flux
+
+/// \cond
+class DataVector;
+/// \endcond
+
+// IWYU pragma: no_forward_declare Tensor
+
+namespace RadiationTransport {
+namespace M1Grey {
+
+/// Implementation of the M1 fluxes for individual neutrino species
```

Comment:
We don't render dox for `detail` namespaces. I'm perfectly happy for you to keep the comment as a C++ comment (`//` instead of `///`)